### ISSUE TITLE
closed interval with countably many origins

### ISSUE THREAD

**Issue by Moniker1998** [pi-base/data#675]
Hello, I've recently been browsing pi-base, and could not find a good example of a space that would be pseudocompact, not compact, $T_1$, and exhaustible by compact sets.

The space I am addressing in this issue is such example, see this post https://math.stackexchange.com/a/4935609/476484

**Comment by prabau** [comment]
If you want to write a pull request for it, that would be great.

**Comment by Moniker1998** [comment]
I forgot how to do this, and additionally I'm going to be off for a week, but if no one is going to write a pull request then I will try to do it.

**Comment by prabau** [comment]
This is a variation on [Line with countably many origins](https://topology.pi-base.org/spaces/S000084), modified so that the part away from the origins becomes compact, to get pseudocompactness.  You could modify your example to be a circle with countably many origins, so that it also becomes [Locally n-Euclidean](https://topology.pi-base.org/properties/P000123) (non-Hausdorff one-dimensional manifold).

**Comment by GeoffreySangston** [comment]
This is also an analog to [Telophase topology S65](https://topology.pi-base.org/spaces/S000065). I support adding it.
I also support changing Telophase topology so that its definition makes it a subspace of line with two origins.

**Comment by GeoffreySangston** [comment]
@Moniker1998 I'll to think about that. If the examples serve equal purposes then I would guess the locally Euclidean example, but I'll see if I can come up with any reason to prefer one or the other, or justify both.

**Comment by Moniker1998** [comment]
Thank you, I'll wait for what @prabau has to say, and add all three examples if no one sees any issue with that.

**Comment by prabau** [comment]
Yeah, I think the two variations of a circle should fine.  And I think we can skip the unit interval with countably many origins.

**Comment by prabau** [comment]
@GeoffreySangston Like you suggested, it would be good to mention that the Telophase topology is a subspace of the line with two origins.

**Comment by Moniker1998** [comment]
@prabau @GeoffreySangston A circle technically doesn't have an origin, so would calling it a circle with countably many origins be fine?

**Comment by prabau** [comment]
Good point. What are synonyms in use for "line with two origins" for comparison?

"Circle with a doubled point" for the two point case?
Not sure about the case of countably many.

**Comment by GeoffreySangston** [comment]
@Moniker1998 I think it's an issue too. As @prabau suggests, 'Circle with a doubled point' works for that one, as in [Everywhere doubled line](https://topology.pi-base.org/spaces/S000086). It's problematic because I'm also not sure what to call the countable variant.

I think I'd prefer not to use 'origin' for the circle, unless we find a reference which does or cannot think of a better name which works for both spaces. It does have the benefit of signaling that the space is similar to the line with two / countably more origins.

Haefliger and Reeb define the concept in general in Definition 3 on page 110 of [their paper](https://www.e-periodica.ch/digbib/view?pid=ens-001%3A1957%3A3%3A%3A215#215). They write *point de branchement*. [Sorcar's translation](https://arxiv.org/pdf/2208.11193) of Reeb + Haefliger's paper (approved by Haefliger apparently) translates this as *branch point* in definition 3. The term 'branch point' seems to appear twice in David Gauld's book on non-metrisable manifolds, with this usage. It's not ideal since "branch point" has more famous meanings.

**Comment by prabau** [comment]
So according to Haelfinger-Reeb definition, each origin of the line with two origins is a "point de branchement", right?  It's not just for spaces like the [branching line](https://en.wikipedia.org/wiki/Non-Hausdorff_manifold#Branching_line), which has a more real branching.

In any case, given the few references you found with that name, I would be ok with "circle with ...-origins".  One advantage is that is related direct to the corresponding "line" spaces.

**Comment by prabau** [comment]
@GeoffreySangston What to you think of the argument:
 circle = reals modulo integers. We can treat the origin on the reals as its equivalence class in the circle, so it kind of makes some sense to call $1=e^0$ the "origin" of the circle in that manner.

**Comment by GeoffreySangston** [comment]
> So according to Haelfinger-Reeb definition, each origin of the line with two origins is a "point de branchement", right? It's not just for spaces like the [branching line](https://en.wikipedia.org/wiki/Non-Hausdorff_manifold#Branching_line), which has a more real branching.

Yes. Sorcar writes that a *branch point* $p \in X$ in a locally Euclidean space $X$ is a point for which there exists another point $q$ such that $p$ and $q$ are not separated by disjoint neighborhoods. Looking at Haefliger-Reeb, I think I confirmed that is what they define it as using my extremely limited understanding of French.


> In any case, given the references you found with that name, I would be ok with "circle with ...-origins". One advantage is that it relates directly to the corresponding "line" spaces.

I support "circle with ... origins".

> @GeoffreySangston What to you think of this: circle = reals modulo integers. We can treat the origin on the reals as its equivalence class in the circle, so it kind of makes some sense to call 1 = e 0 the "origin" of the circle in that manner. (and "origin" = identity element of the additive group of reals = identity element of the circle (as multiplicative group of complexes of modulus 1)

I deliberated with myself for too long now and I think it's okay. Will we refer to the 'origins' of the space in the trait files?

**Comment by GeoffreySangston** [comment]
I'm kind of curious if there's a locally Euclidean space such that the [Hausdorff reflection](https://mathoverflow.net/questions/78175/largest-hausdorff-quotient/78200#78200) involves more than 1 step of gluing the non-separated (branch) points. Maybe one of the complicated examples from Haefliger-Reeb (Sorcar)?

**Comment by prabau** [comment]
The one-point compactification of the line is a circle.  Similarly, the "circle with $\alpha$ origins" is the one-point compactification of the "line with $\alpha$ origins".  (Something that could be mentioned in the description.)  But it's another good reason to go with that name.

**Comment by GeoffreySangston** [comment]
> Thank you, I'll wait for what @prabau has to say, and add both circle with two origins and circle with countably many origins, if no one has an issue with that (the justification for [ 0 , 1 ] with countably many origins is indeed pretty weak)

Do you still want to add these @Moniker1998? If you're working on other things, I don't mind going ahead and adding them with a few traits to get them started.

**Comment by Moniker1998** [comment]
@GeoffreySangston feel free to
